Dockerfile

```diff
@@ -6,4 +6,4 @@ COPY . ./
 
 RUN swift package resolve
 RUN swift package clean
-CMD swift test --parallel
+CMD swift test --parallel -v
```

Comment:
Do we wanna try the auto discovery stuff on Swift 5.1? Maybe separate PR?

Reply:
yeah I think separate PR